assets/js/blocks/cart-checkout/subscriptions-integration.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import {
+	ExperimentalOrderShippingPackages,
+	ShippingRatesControlPackage,
+} from '@woocommerce/blocks-checkout';
+import { registerPlugin } from '@wordpress/plugins';
+import { useStoreCart, useSelectShippingRate } from '@woocommerce/base-hooks';
+import { useMemo } from '@wordpress/element';
+
+const RenderSubscriptionPackages = () => {
+	const {
+		extensions: { subscriptions },
+	} = useStoreCart();
```

Comment:
I didn't fully test this pattern yet, but I think we can have `RenderSubscriptionPackages` in a higher level SlotFill (CartCheckout maybe?) that passes down `cart` data instead of using useStoreCart.
If it's not possible, we can expose a new hook that only pass down relevant data we need, but we would need to remove `fillProps` to not have to implementation of the same thing.

Reply:
Are there use cases for exposing full cart hook? What is dangerous to expose exactly?